NuRadioReco/framework/base_station.py

```diff
@@ -198,6 +198,16 @@ def set_electric_fields(self, electric_fields):
     def get_electric_fields(self):
         return self._electric_fields
 
+    def get_electric_field_ids(self):
+        """
+        returns a list with the electric field IDs of all simElectricFields of the simStation
+        """
+        efield_ids = []
+        for efield in self._electric_fields:
+            efield_ids.append(efield.get_unique_identifier())
+        efield_ids.sort()
```

Comment:
I would not necessarily expect this sort here. Maybe add it to the information to the doc string